src/o365/spo/commands/feature/feature-enable.ts

```diff
@@ -0,0 +1,154 @@
+import { ContextInfo } from '../../spo';
+import request from '../../../../request';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption,
+  CommandValidate,
+} from '../../../../Command';
+import SpoCommand from '../../../base/SpoCommand';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  url: string;
+  featureId?: string;
+  scope?: string;
+  force?: boolean;
+}
+
+class SpoFeatureEnableCommand extends SpoCommand {
+  public get name(): string {
+    return commands.FEATURE_ENABLE;
+  }
+
+  public get description(): string {
+    return 'Enable feature for the specified site or web';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.featureId = (!(!args.options.featureId)).toString();
+    telemetryProps.url = (!(!args.options.url)).toString();
+    telemetryProps.scope = (!(!args.options.scope)).toString();
+    telemetryProps.force = args.options.force || false;
+    return telemetryProps;
+  }
+
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: (err?: any) => void): void {
+    let scope = args.options.scope;
+    let force = args.options.force;
+    
+    if(!scope) {
+      scope = "web";
+    }
+    if(!force){
+      force = false;
+    }
+
+    if (this.verbose) {
+      cmd.log('Retrieving request digest');
+    }
+    this.getRequestDigest(args.options.url)
+        .then((contextInfo: ContextInfo):Promise<string> => {
+          if (this.verbose) {
+            cmd.log(`Enabling feature '${args.options.featureId}' on scope '${scope}' for url '${args.options.url}' (force='${force}')...`);
+          }
+  
+          let url = `${args.options.url}/_api/${scope}/features/add(featureId=guid'${args.options.featureId}',force=${force})`;
+          const requestOptions: any = {
+            url: url,
+            headers: {
+              accept: 'application/json;odata=nometadata',
+              'X-RequestDigest': contextInfo.FormDigestValue,
+            },
+          };
+  
+          return request.post(requestOptions);
+        })
+        .then((res: string): void => {
+          if (this.verbose) {
+            cmd.log('Feature activated');
```

Comment:
Most of the commands have `cmd.log(vorpal.chalk.green('DONE'));` as finishing message in verbose. Just a suggestion is that we can replace `cmd.log('Feature activated');` with  `cmd.log(vorpal.chalk.green('DONE'));` for consistency.

Reply:
Fixed